### PR TITLE
Feature/registration

### PR DESCRIPTION
**Pull request by AzizaHelmy**
Login and Register Screens

### CODE REVIEW

**Comment by AzizaHelmy** [comment]
@alihrhera 
please review the changes

File: app/src/main/java/com/example/mensfashion/ui/details/DetailsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class DetailsFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```
Comment (alihrhera) [inline app/src/main/java/com/example/mensfashion/ui/details/DetailsFragment.kt:18]: you don't need onCreateView any more

File: app/src/main/java/com/example/mensfashion/ui/favorites/FavoritesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import com.example.mensfashion.R
+
+class FavoritesFragment : Fragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
```
Comment (alihrhera) [inline app/src/main/java/com/example/mensfashion/ui/favorites/FavoritesFragment.kt:17]: you don't need onCreateView any more

File: app/src/main/java/com/example/mensfashion/ui/login/LoginFragment.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        binding.btnLogin.setOnClickListener {
+            email = binding.etEmail.text.toString()
+            passward = binding.passward.text.toString()
+            loginViewModel.userLogin(email, passward)
+            loginViewModel.loginResult.observe(viewLifecycleOwner) { result ->
+                when (result) {
+                    "invalid Email!" -> binding.layoutEmail.error = "invalid Email!"
+                    "required"->binding.layoutPass.error= "required"
+                    else->{
+                        findNavController().navigate(R.id.action_loginFragment2_to_homeFragment)
```
Comment (alihrhera) [inline app/src/main/java/com/example/mensfashion/ui/login/LoginFragment.kt:44]: you can use NavigationExtention like this 
 //            navigateTo( destination-id )
no need to 
   findNavController().navigate(R.id.action_loginFragment2_to_homeFragment)

File: app/build.gradle [hunk 4]
```diff
@@ -43,10 +43,16 @@ dependencies {
...
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.5.0'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.5.0'
+    implementation 'androidx.security:security-crypto-ktx:1.1.0-alpha03'
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
     implementation "androidx.security:security-crypto:1.1.0-alpha01"
     implementation "com.google.code.gson:gson:2.9.0"
 
+    //circule Img
```
Comment (alihrhera) [inline app/build.gradle:56]: we don't need circule image handeler 
we can handel it using image load lib like picasso or glid 
or hndel from XML  using ShapeableImageView witch it a material componant

**Review by alihrhera — COMMENTED** [review]
please review my comments 
and please change the design to be like the attached image

File: app/src/main/java/com/example/mensfashion/ui/details/DetailsFragment.kt (hunk 1 above)
Comment (AzizaHelmy) [reply app/src/main/java/com/example/mensfashion/ui/details/DetailsFragment.kt:18]: yes, it was  an initial code, I forget to remove it after using Basefragment on the other screen

File: app/src/main/java/com/example/mensfashion/ui/favorites/FavoritesFragment.kt (hunk 2 above)
Comment (AzizaHelmy) [reply app/src/main/java/com/example/mensfashion/ui/favorites/FavoritesFragment.kt:17]: yes, it was  an initial code, I forget to remove it after using Basefragment on the other screen

File: app/src/main/java/com/example/mensfashion/ui/login/LoginFragment.kt (hunk 3 above)
Comment (AzizaHelmy) [reply app/src/main/java/com/example/mensfashion/ui/login/LoginFragment.kt:44]: ok, will use it ISA

File: app/build.gradle (hunk 4 above)
Comment (AzizaHelmy) [reply app/build.gradle:56]: thanks for the tip, will use one of them ISA

**Pull request merged**